app/views/nominations/nominate_induction_coordinator/start.html.erb

```diff
@@ -1,14 +1,20 @@
-<h1>Nominate an induction tutor</h1>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">Nominate an induction lead or tutor</h1>
 
-<%= render partial: 'nominations/school_details', locals: { school: @nominate_induction_tutor_form.school, title: "Your school" } %>
```

Comment:
Is this partial still used?

Reply:
Yes, in a review screen. Adding another use